src/utils/set-deep.ts

```diff
@@ -26,10 +27,6 @@ function isValidKey(key: unknown) {
   return key !== '__proto__' && key !== 'constructor' && key !== 'prototype';
 }
 
-function isObject(val: unknown) {
-  return val !== null && (typeof val === 'object' || typeof val === 'function');
-}
```

Comment:
Yep!  Not sure why we had the function check.  As long as all the tests pass!  (Don't have CI setup yet)